uobtheatre/productions/models.py

```diff
@@ -445,6 +442,21 @@ def total_capacity(self, seat_group=None):
         response = self.performance_seat_groups.aggregate(Sum("capacity"))
         return response["capacity__sum"] or 0
 
+    def total_capacity(self, seat_group=None):
```

Comment:
This should be a propety probably, and it as such it probably shouldn't accept a seat group argument. There are places where this was used - this will need to be refactored to use the new seat group capacity func